### ISSUE TITLE
No label text on cpp(mac) target

### ISSUE THREAD

**Issue by varadig** [BowlerHatLLC/feathersui-openfl#38]
While compile to html5 target works great, but on mac target the label of the button componenet is gone.

https://www.dropbox.com/s/w25qfu89ee8nr5y/Screenshot%202020-05-24%2010.19.27.png?dl=0

```
import feathers.events.TriggerEvent;
import feathers.layout.AnchorLayoutData;
import feathers.layout.AnchorLayout;
import feathers.controls.Button;
import feathers.controls.Application;

class Main extends Application {
	private var button:Button;

	public function new() {
		super();

		trace("Hello, world!");

		this.layout = new AnchorLayout();

		this.button = new Button();
		this.button.addEventListener(TriggerEvent.TRIGGER, (event) -> {
                       trace("Hello World");
		});

		this.button.layoutData = AnchorLayoutData.center();
		this.addChild(this.button);
		this.button.text = "Hello";
	}
}

```

**Comment by varadig** [comment]
The Label control isn't display text as well

**Comment by varadig** [comment]
on windows target is no problem with the label:
https://www.dropbox.com/s/u52sx36fp6li180/Screenshot%202020-05-25%2006.28.16.png?dl=0

**Comment by joshtynjala** [comment]
openfl/openfl#2316

Fixed in openfl/openfl@8e3f356ebfe388318a9cfe04d573e04e8828a3c6. This commit should be included in the next update to OpenFL.

**Comment by varadig** [comment]
Oh, I missed this. Thank you @joshtynjala
